tiledb/sm/array_schema/array_schema.cc

```diff
@@ -1038,6 +1051,42 @@ Status ArraySchema::set_cell_validity_filter_pipeline(
   return Status::Ok();
 }
 
+void ArraySchema::set_dimension_label_filter_pipeline(
+    const std::string& label_name, const FilterPipeline& pipeline) {
+  auto& dim_label_ref = dimension_label_reference(label_name);
+  if (!dim_label_ref.has_schema()) {
+    throw ArraySchemaStatusException(
+        "Cannot set filter pipeline for dimension label '" + label_name +
+        "'; That dimension label does not have a stored schema.");
```

Comment:
Is the issue here that the DL does not _have_ a stored schema, or that the referenced schema has not yet been loaded? When/how is the loading triggered?

Reply:
👍 please re-word in one of the upcoming PRs. Marking resolved - thanks.